packages/connectivity/connectivity/lib/connectivity.dart

```diff
@@ -23,12 +23,12 @@ class Connectivity {
     if (_singleton == null) {
       _singleton = Connectivity._();
     }
-    return _singleton;
+    return _singleton!;
```

Comment:
Interesting. In some cases the analyzer is smart enough to figure out that `_singleton` cannot be `null` at this point, so `!` isn't required.

Reply:
Got an error trying to remove !
```
./lib/connectivity.dart:25:12: Error: A value of type 'Connectivity?' can't be returned from a function with return type 'Connectivity'.
 - 'Connectivity' is from 'package:connectivity/connectivity.dart' ('../lib/connectivity.dart').
    return _singleton;```